lib/jsdom/living/window/History-impl.js

```diff
@@ -11,8 +10,6 @@ exports.implementation = class HistoryImpl {
     this._actAsIfLocationReloadCalled = privateData.actAsIfLocationReloadCalled;
     this._state = null;
     this._latestEntry = null;
```

Comment:
_state and _latestEntry seem like prime candidates for moving into SessionHistory, although I haven't looked at the spec that much recently.